src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/payment.html.twig

```diff
@@ -0,0 +1,24 @@
+{% extends '@SyliusShop/Layout/checkout.html.twig' %}
+
+{% form_theme form 'SyliusUiBundle:Form:theme.html.twig' %}
+
+{% block content %}
+    {% include '@SyliusShop/Checkout/_steps.html.twig' %}
```

Comment:
According to recent changes, you could use ``{% include '@SyliusShop/Checkout/_steps.html.twig' with { 'active': 'payment' } %}`` to display step buttons properly ;)